skyyrose-suite/plugins/skyyrose-market/skills/skyyrose-social-social-listening-plan/SKILL.md

```
@@ -136,10 +136,9 @@ Common variations and misspellings:
 
 ### 3. Tool Configuration
 
-```
-## Tool Stack
+#### Tool Stack
 
-### Free (always-on baseline)
+##### Free (Always-On Baseline)
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)